source/process/developer.rst

```diff
@@ -2,123 +2,123 @@
 
-Working in open source means your work is publicly visible. Your code will receive both credit and critique from the community and with the right mindset and support that can lead to you making the best engineering decisions of your career. 
+Working in open source means your work is publicly visible. Your code will receive both credit and critique from the community and with the right mindset and support that can lead to you making the best engineering decisions of your career.
 
 Core committers include volunteer developers from the community, staff by organizations deploying and investing in Mattermost, as well as staff paid by Mattermost, Inc.
 
-This document summarizes the resposibilities and requirements of core committers employed by Mattermost, Inc., skill level requirements and hiring process. Staff developers work closely with Mattermost product management to envision, design, plan, develop, coordinate, review and merge improvements per our `contribution guidelines <https://docs.mattermost.com/developer/contribution-guide.html>`_ on a regular basis. 
+This document summarizes the responsibilities and requirements of core committers employed by Mattermost, Inc., skill level requirements and hiring process. Staff developers work closely with Mattermost product management to envision, design, plan, develop, coordinate, review and merge improvements per our `contribution guidelines <../developer/contribution-guide.html>`_ on a regular basis.
 
-Ideally, developers are "full stack", and comfortable moving across backend and frontend projects. As part of our `learn, master, teach <https://docs.mattermost.com/process/training.html#learn-master-teach>`_ mindset, our environment has people consistently cross-training, becoming experts and passing on knowledge to new team members rotating into new areas of responsibility. That said, we also welcome engineers who prefer to specialize in backend and frontend, since there's ample projects to find in both areas.  
+Ideally, developers are "full stack", and comfortable moving across backend and frontend projects. As part of our `learn, master, teach <training.html#learn-master-teach>`_ mindset, our environment has people consistently cross-training, becoming experts and passing on knowledge to new team members rotating into new areas of responsibility. That said, we also welcome engineers who prefer to specialize in backend and frontend, since there's ample projects to find in both areas.
 
 Responsibilities
 -------------------------
 
 - Develop features from proposal to polished end result.
-- Support and collaborate with our community service engineers to find the root cause of issues, find, deliver and document effective solutions. 
+- Support and collaborate with our community service engineers to find the root cause of issues, find, deliver and document effective solutions.
```

Comment:
Actually, that whole sentence doesn't make sense to me. However, because it's an existing one and not related to the purpose of this PR, I propose creating a new PR for this to review the entire doc.